ginga/Control.py

```diff
@@ -1503,7 +1503,7 @@ def _load_n_switch(imname, path, image_future):
                 # perpetuate the image_future
                 image.set(image_future=image_future, name=imname,
                           path=path)
-                self.fv.gui_do(self.switch, image)
+                self.fv.gui_do(self.switch_image, image)
```

Comment:
Good catch!  But I think this is actually supposed to be `_switch` (the function that is defined above).